Controllers/CouriersController.go

```diff
@@ -26,8 +26,22 @@ func (h *Handler) GetOneCourier(c *gin.Context) {
 	}
 	Courier, err = h.services.GetCourier(id)
 	if err != nil {
-		c.JSON(http.StatusBadRequest, gin.H{"No such courier": err})
+		c.JSON(http.StatusInternalServerError, gin.H{"No such courier": err})
 		return
 	}
 	c.JSON(http.StatusOK, Courier)
 }
```

Comment:
It is not server error. I think that it is 400

Reply:
fixed